src/templates/search.html

```diff
@@ -92,7 +91,7 @@ <h3 id='dateFormats'>Date formats</h3>
 
 
 <!-- SEARCH PAGE BEGIN -->
-<form name='search' method='get' action='search.php'>
+<form id='search' name='search' method='get' action='search.php'>
```

Comment:
why is an id needed here? Also, it's too generic. Maybe `mainSearchForm` would be better, only if really needed ofc.

Reply:
Yes, that needs to be removed. I should use the submit button to trigger the form submission.